spec/services/food_service_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe FoodService do
+    it 'gets data from api' do
+        foods = FoodService.search_food('cheese') 
+ 
+        expect(foods).to be_a(Hash)      
```

Comment:
We'd expect to see more assertions here around the presence of certain keys that will then be used in the PORO